File: dev-docs/snippets/users/get-basic.swift
import JsBaoClient

// Fetch one user's basic public profile by id.
func getBasic(client: JsBaoClient, userId: String) async throws {
  // #region example
  let user: BasicUserInfo = try await client.users.getBasic(userId: userId)
  let name = user.name
  // #endregion example
  _ = name
}

File: dev-docs/snippets/users/get-basic.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Fetch one user's basic public profile by id.
export async function getBasic(client: JsBaoClient, userId: string) {
  // #region example
  const user = await client.users.getBasic(userId);
  // #endregion example
  return user;
}
